extensions/positron-python/src/client/positron/webAppContexts.ts

```diff
@@ -6,7 +6,18 @@
 import * as vscode from 'vscode';
 import { executeCommand } from '../common/vscodeApis/commandApis';
 
-const libraries: string[] = ['streamlit', 'shiny', 'dash', 'gradio', 'flask', 'fastapi'];
+function getSupportedLibraries(): string[] {
+    const libraries: string[] = ['streamlit', 'dash', 'gradio', 'flask', 'fastapi'];
+
+    const shinyExtensionId = 'Posit.shiny';
+    const shinyExtension = vscode.extensions.getExtension(shinyExtensionId);
+
+    // if shiny extension is installed, we don't need to handle it
+    if (shinyExtension) {
```

Comment:
if it is disabled, `shinyExtension` will be undefined, so I'm no longer checking if it is active. this should fix the double dropdown!